add(tem): dmarc documentation pages (#2800)
scaleway/docs-content: managed-services/transactional-email/how-to/add-dmarc-record.mdx

---
meta:
  title: How to configure a DMARC record
  description: This page explains how to configure a DMARC record
content:
  h1: How to configure a DMARC record
  paragraph: This page explains how to configure a DMARC record

> tags: spf spf-record dkim dkim-records transactional

Reviewer: ```suggestion
tags: spf spf-record dkim dkim-records dmarc dmarc-record transactional
```
Are we using "-record" or "-records"?

Author: we can use everything!

dates:
  validation: 2024-02-21
  posted: 2024-02-21
categories:
  - managed-services
---

This page shows you how to configure a [DMARC record](/managed-services/transactional-email/concepts/#dmarc-record) on your domain from the **Scaleway console**.
If you have registered a domain at another provider, you can still use this documentation, as the process is similar.

<Macro id="iam-requirements" />

<Message type="requirement">
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/).
  - You have a domain name.
  - You have [configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).

>   - You have configured SPF and DKIM records on your domain.

Reviewer: The DMARC record will be added as long as SPF and DKIM records. I think we can remove this line.

</Message>

1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu.

> 2. Click your domain. You are redirected to the records configuration page.

Reviewer 2: ```suggestion
2. Click your domain name. You are redirected to the records configuration page.
```

> 3. Add a `TXT` record type in the root zone of your domain on your DNS zone.

Reviewer: ```suggestion
3. Add a `TXT` record type in the same zone of your TEM domain on your DNS zone.
```

The domain registered at TEM is not necessarily in the root zone. 

For example, I have "mydomain.tld". And a zone "tem", I can register a TEM domain "tem.mydomain.tld" or "stg.tem.mydomain.tld".
In the first case, I'm in the root record of the "tem" zone. In the second case, I'm not a root record on the "tem" zone.
Note that this line is also in another tem how-to

4. Enter `_dmarc` in the **Name** field, followed by the domain name. For example, if your domain is called `my-domain.com`, you should have `_dmarc.my-domain.com.` in the **Name** field.
5. Enter your desired tags and their values in the **Value** section. Make sure that you add the mandatory `p` and `v` tags, and that you separate tags from each other with semi-colons. Find out about DMARC configuration options in our [dedicated documentation](/managed-services/transactional-email/reference-content/dmarc-configuration/).
    Your **Value** field should look similar to the following:
    ```
    v=DMARC1; p=none; rua=mailto:your@email.com
    ```
6. Click **Add record**.

<Message type="tip">
  For more in-depth information about DMARC configuration, refer to the [dedicated documentation](/managed-services/transactional-email/reference-content/dmarc-configuration/) page.
</Message>

> 

Reviewer: A note could be added: You can have only one _dmarc TXT records per `Name`
Example: `_dmarc.mydomain.tld` and `_dmarc.tem.mydomain.tld` is valid. but not 2 `_dmarc.mydomain.tld`

2. Click your domain. You are redirected to the records configuration page.
3. Add a `TXT` record type in the same zone as your Transactional Email's domain, on your DNS zone.
4. Enter `_dmarc` in the **Name** field, followed by the domain name. For example, if your domain is called `my-domain.com`, you should have `_dmarc.my-domain.com.` in the **Name** field.
5. Enter your desired tags and their values in the **Value** section. Make sure that you add the mandatory `v` and `p` tags, and that you separate tags from each other with semi-colons. Find out about DMARC configuration in our [dedicated documentation](/managed-services/transactional-email/reference-content/dmarc-configuration/).
    Your **Value** field should look similar to the following
    ```
    v=DMARC1; p=none; rua=mailto:your@email.com
    ```
    <Message type="important">
     It is possible to configure several DMARC records for one domain, but they must have a different name to be valid.
     For example, the `_dmarc.mydomain.com` and `_dmarc.tem.mydomain.com` DMARC records are valid.
    </Message>

> 6. Click **Add record**.

Reviewer 3: I don't do suggestions as I don't know all the documentation writing guidelines and good syntax :)
We cannot manage our domain by the TEM page, I would rather say:
1. Click **Domains and DNS** in the **Network** section of the [console](https://console.scaleway.com/) side menu.
2. Click your domain then **DNS Zones** then click the zone where you want to add your DMARC record .

Maybe we can add a Note to say that it is only valid for domains managed by Scaleway. Steps could be different for other providers.

<Message type="important">
  Emails sent from sources not configured in your SPF record, or that do not have a DKIM **will be rejected if you use the `p=reject` policy**.

> </Message>

Reviewer 3: I don't actually get why we add this note here, as we don't talk about "p=reject" before.
I would rather see it on the "Understanding DMARC configuration" (where is already is :) )
Or let it here but make it in accordance with the example record " v=DMARC1; p=none; rua=mailto:your@email.com" mentioned above.
